tests/test_core.py

```diff
@@ -6680,7 +6680,10 @@ def encode_utf8(data):
         else:
           return data
 
-      data = encode_utf8(json.load(open(map_filename, 'r')))
+      data = json.load(open(map_filename, 'r'))
+      if str is bytes:
```

Comment:
could this be `asstr`?

Reply:
`data` is not a string but a dictionary so it can't.

Reply:
oh, right, thanks.